github/github.go

```diff
@@ -986,6 +996,59 @@ func (g *GitHub) UpdateReleasePage(
 	return release, nil
 }
 
+// toRepositoryRelease builds a repository release from the set of options
+func (urpo *UpdateReleasePageOptions) toRepositoryRelease() *github.RepositoryRelease {
+	request := &github.RepositoryRelease{}
+
+	if urpo.Name != "" {
+		request.Name = &urpo.Name
+	}
+	if urpo.Body != "" {
+		request.Body = &urpo.Body
+	}
+
+	request.Draft = &urpo.Draft
+	request.Prerelease = &urpo.Prerelease
+
+	if urpo.Latest {
+		request.MakeLatest = ptr.To("true")
+	} else {
+		request.MakeLatest = ptr.To("false")
+	}
+
+	return request
+}
+
+// UpdateReleasePageWithOptions updates release pages (same as UpdateReleasePage),
+// but does so by taking a UpdateReleasePageOptions parameter. It will _not_ set
+// a release as latest unless the corresponding option is set.
+func (g *GitHub) UpdateReleasePageWithOptions(owner, repo string,
+	releaseID int64,
+	tag, commitish string,
```

Comment:
I'm wondering if we should move those parameters to the options struct as well. 🤔

Reply:
Everything in the options is, well, optional. tag and target comitish are not (tag is is [required by the API](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28), and honestly, without `commitish` this doesn't make much sense, so it's sort of "required" as well).